libraries/stdlib/js/src/js/utf8.js

```diff
@@ -0,0 +1,64 @@
+// http://jonisalonen.com/2012/from-utf-16-to-utf-8-in-javascript/
+String.prototype.toUTF8Array = function() {
```

Comment:
I don't know where to put it :) Which global namespace do you recommend?

Reply:
It could be added to some file inside `libraries/stdlib/js/src/js` or reimplemented in Kotlin. But before that, would be nice to decide do we need it in stdlib or not.

cc @ilya-g @cy6erGn0m